lib/data-atom-controller.js

```diff
@@ -303,6 +304,28 @@ export default class DataAtomController {
   }
 
   /**
+   * Query table
+   */
+  queryTable(el) {
+    if (this.isEditorNotActive())
+      return;
+
+    if (!this.mainView.isShowing)
+      this.showMainView();
+
+    var query = 'SELECT * FROM ' + el.target.innerText + ' LIMIT 100';
```

Comment:
This PR is a great addition. But I'd rather not have SQL code generated in the main controller. We need to move this into the providers (https://github.com/lukemurray/data-atom/tree/master/lib/data-managers) or some other abstraction.

This allows the different DBs we support to implement Query Table differently. I know right now, they all support the same SQL for that, but if we supported MongoDB or others it would generated non-SQL commands.